client/src/components/app.jsx

```diff
@@ -28,11 +27,9 @@ class App extends React.Component {
     this.handleHeaderClick = this.handleHeaderClick.bind(this);
     this.handleExitClick = this.handleExitClick.bind(this);
   }
-  //'http://localhost:3005/restaurants/52566848/photos'
-  ///api/restaurants/${window.location.pathname}/photos
-
+  
   componentDidMount() {
-    axios.get(`http://localhost:3005/restaurants/52566848/photos`).then((data) => {
+    axios.get(`/api/restaurants/${window.location.pathname}/photos`).then((data) => {
```

Comment:
This might need to have substring logic to only get the restaurant ID.. we are targeting it as: /restaurants/${restaurantId} in the browser, so pathname might include /restaurants/ as well as the id

Reply:
I see, I will make those changes right now!